### PR TITLE
docs: update README

### PR DESCRIPTION
**Pull request by matthewlilley**
Add installation instructions, further configuration instructions, and a disclaimer.

### CODE REVIEW

File: example/channel-bot.toml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+# Give the channel a name to make them easily identifiable in the notifications
+alias = "wumbo-boltz"
+# Channel ID of the channel
+channelid = 619899158240231424
+
+# The ratio is calculated with: local balance / channel capacity
+# The minimal ratio before the channel is considered imbalanced
+minratio = "0.1"
+# The maximal ratio before the channel is considered imbalanced
+maxration = "0.6"
```
Comment (michael1011) [inline example/channel-bot.toml]: Missing empty new line at the end of the file

File: README.md [hunk 2]
```diff
@@ -26,55 +26,38 @@ More to come...
...
-# The maximal ratio before the channel is considered imbalanced
-maxration = "0.6"
+The channel cleaner takes care of force closing [zombie channels](https://medium.com/@gcomxx/get-rid-of-those-zombie-channels-1267d5a2a708). The interval at which the channels should be checked for zombies and the number of days that are needed for a channel to become a zombie are [configurable](#configuration).
+
+## Installation
+
+```shell
+git clone https://github.com/BoltzExchange/channel-bot.git
+
+cd channel-bot
```
Comment (michael1011) [inline README.md]: I would prefer having the repository cloned into the GOPATH:
```suggestion
go get -d github.com/BoltzExchange/channel-bot
cd $GOPATH/src/github.com/BoltzExchange/channel-bot
```

**Review by michael1011 — CHANGES_REQUESTED** [review]
Just two minor nits. Can be merged afterwards. Thank you very much! :+1:

File: example/channel-bot.toml (hunk 1 above)
Comment (matthewlilley) [reply example/channel-bot.toml]: amended

File: README.md (hunk 2 above)
Comment (matthewlilley) [reply README.md]: amended

**Review by michael1011 — APPROVED** [review]

**Pull request merged**